routing/route1.config

```diff
@@ -0,0 +1,38 @@
+Number of Nodes:
```

Comment:
Make config a `.yml` file. Will be a lot easier to read and remove a lot of your parsing code.

Reply:
You will need PyYaml installed to read these files. Given that it is a installed package it should be included in your Dockerfile